RELEASE

```diff
@@ -1,22 +1,30 @@
 *******************************************************************************
 
-Axom: ................................, version 0.5.0
+Axom: ................................, version 0.6.0
 
 Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC. 
 Produced at the Lawrence Livermore National Laboratory.
 All rights reserved. See details in the file axom/LICENSE.
 
+Current open source release information:
+Unclassified/Open Source - BSD Distribution
+LLNL-CODE- 825966
```

Comment:
```suggestion
LLNL-CODE-825966
```

Reply:
D'oh! Good catch. Fixed.  @agcapps my PR description may have been confusing. Please approve the other one that merges this into main. This one is just for convenient reviewing.